sherlock_project/resources/data.json

```diff
@@ -1082,6 +1082,13 @@
     "urlMain": "https://hudsonrock.com",
     "username_claimed": "testadmin"
   },
+  "HuggingFace": {
+    "errorMsg": "status_code",
+    "errorType": "message",
```

Comment:
Looks like you made typo here. You mentioned that the site returns a 200 status code if the username exists. Therefore, you only need `"errorType": "status_code"`